File: Development/ALL Android Projects/YourBudget Project/Ofppt/Project Mobile/Android Project Code/YourBudget/app/src/main/java/com/example/yourbudget/MainActivity.java
package com.example.yourbudget;

import androidx.appcompat.app.AppCompatActivity;
import androidx.core.content.ContextCompat;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    Button connexion;
    Button Inscrie;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        connexion = findViewById(R.id.button);
        Inscrie = findViewById(R.id.button2);
//        getWindow().getStatusBarColor(ContextCompat.getColor(MainActivity.this,R.color.ic_launcher_background));

        connexion.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(getApplicationContext(),Login.class);
                startActivity(intent);
                finish();


            }
        });
        Inscrie.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent intent = new Intent(getApplicationContext(),Register.class);
                startActivity(intent);
                finish();


            }
        });
    }
}

File: Development/ALL Android Projects/Android/IntentsIntentFilters/app/src/main/java/com/example/intentsintentfilters/MainActivity.kt
package com.example.intentsintentfilters

import android.annotation.SuppressLint
import android.content.ActivityNotFoundException
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class MainActivity : AppCompatActivity() {

    var button:Button?= null;
    var textView:TextView?= null;
    var textView2:String?= null;

//    var button1=null
//    var textview1=null



    @SuppressLint("SuspiciousIndentation")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        button= findViewById<Button>(R.id.button);
        textView= findViewById<TextView>(R.id.textView);




        button!!.setOnClickListener(){

            val theApp = "com.google.android.youtube"
            textView!!.text="We're here for you and your loved ones!"
//            textView!!.

//            textView= textView!!.freezesText=""
//            textView= textView!!.textDirection=""
//            textView= textView!!.textLocale=""
//            textView= textView!!.textColors=""
//            textView= textView!!.textScaleX=""

                Intent(Intent.ACTION_MAIN).also {
                it.`package`=theApp

            try {
                startActivity(it)
            } catch (ex : ActivityNotFoundException){
                ex.printStackTrace()
            }



            }

        }

//        button!!.setOnClickListener()

    }
}